terraform/modules/data_share_processor/data_share_processor.tf

```diff
@@ -449,28 +449,11 @@ output "aggregate_queue" {
 }
 
 output "specific_manifest" {
```

Comment:
What do we use this output for at all anymore? So that we can provide the default specific manifest template to `key-rotator`?

Reply:
Yep: `key-rotator` uses this as the "template" from which it can generate full manifests for (locality, ingestor) pairs which do not already have one. [I did this to make sure the generated manifest matches as closely as possible to what was being generated by `deploy-tool`.]